dlt/common/destination/reference.py

```diff
@@ -561,6 +617,18 @@ def should_truncate_table_before_load_on_staging_destination(self, table: TTable
         return True
 
 
+class WithReadableRelations(ABC):
+    """Add support for getting readable reletions form a destination"""
+
+    @abstractmethod
+    def get_readable_relation(
```

Comment:
I think you should split into `table_relation` and `query_relation`

Reply:
this is done